### DISCUSSION TITLE
Install instructions for BasicTeX

### DISCUSSION THREAD

**Discussion by majid-vaghari** [tex-persian/xepersian#3] [Ideas]
Hello,

I've been using XePersian for a while and recently tried to install it on my BasicTeX installation.
BasicTeX is a minimal distribution of TeXLive which does not include most of the packages of CTAN. In my opinion most of those packages are not used for most of practical use cases of LaTeX distributions.

After I tried to install xepersian on this distribution I found out that tlmgr doesn't manage dependencies for packages. So I had to install a few other packages.

I'm writing the list of packages that I installed and I'm asking the maintainers to add those that I couldn't find and maybe add it as a one line command to installation instruction of xepersian.
I found these packages from [texlive-lang-arab](https://packages.debian.org/sid/texlive-lang-arabic) from debian packages

```bash
$ tlmgr update --self
$ tlmgr update --all
$ tlmgr install xepersian xepersian-hm bidi bidihl zref hyphen-farsi persian-bib xindy-persian
```

Hope this helps

**Comment by vafa-khalighi** [comment]
Thanks for contributing this. This is certainly useful and I hope that it can help those users who decide to install BasicTeX. Having said that, I encourage new users to install full MacTeX / TeXLive as you never know which package(s) you are going to need and installing them separately can be a painful experience for a newcomer. Of course if disk space is a concern, then people might want to try your suggestion.
